pages/en/lb4/Defining-and-validating-the-API.md

```diff
@@ -284,7 +284,7 @@ _.merge(spec, CategoryAPI);
 export default spec;
 ```
 
-You can then bind the full spec to the application using `server.spec()`. This is done on the server level, because each server instance can expose a different (sub)set of API.
+You can then bind the full spec to the application using `app.api()`. Normally, this is done on the server level because each server instance can expose a different (sub)set of API, but since `RestApplication` uses only one REST server, you can bind the spec at application level.
```

Comment:
I am proposing a slightly different phrasing:

```markdown
You can then bind the full spec to the application using `app.api()`.
This works well for applications with a single REST server, because
there is only one API definition involved.

If you are building an application with multiple REST servers,
where each server provides a different API, then you need 
to call `server.api()` instead.
```